### DISCUSSION TITLE
#42 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Riely007** [tog-ETH-er/togETHer#19] [General]
Agenda:

Community updates
Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by Thegaram** [comment]
## Crypto Tech Night

06.17: CTN Let's Talk #1
\w @BTCdayu, @forgivenever, @zeroonerobin, @Chris_Defi, @CryptosLaowai
link: https://twitter.com/CryptoTechNight/status/1537712413155291136

06.24: NFT如何走出困局
\w HashKey Me
link: https://twitter.com/hashkey_me/status/1539935513662361606

06.30: Blockchain at Universities -- The Next Generation of Buidlers
\w Tsinghua University, Northwestern University (?)
link: TBD

**Reply by Emilyxiannv** [reply]
厉害了

**Comment by liushooter** [comment]
### Rebase

- [Web3 日报](https://rebase.network/posts/category/web3daily) 正常更新 14 期

- 更新 [多篇招聘](https://github.com/rebase-network/who-is-hiring)
![2022-06-27 at 21 02](https://user-images.githubusercontent.com/1963646/175948097-146dc228-6df3-4d93-a311-47300b13bc05.png)

- 更新多篇 [活动介绍](https://github.com/rebase-network/event-pool)
![2022-06-27 at 21 01](https://user-images.githubusercontent.com/1963646/175947933-7480dfd7-beb6-49a7-a2ca-a2275beffdae.png)

**Comment by winkybupt** [comment]
## 原语里弄
---
1. 原语里弄 2 周年活动

**Comment by bobjiang** [comment]
- csdn以太坊大会
- gitcoin gr14结束 , claim
- gitcoindao year review (translator required) https://gitcoindao.com/yearinreview/yir.pdf

share
- https://andersbrownworth.com/blockchain/

**Comment by Emilyxiannv** [comment]
### DAOSquare
6/19邀请到DAOSquare Summoner @Typto,  Investment Partner of Zonff Partners 的@Aria， Cofounder of MetaNetwork @Minako， member of RedlineDAO @Aylwin，前中国国际广播电台、香港凤凰卫视主持人 HYDAO海盗电台创始人 @加菲众， Investment Director of HashGlobal @Walter
在Twitter上进行了一场关于“Creator Economy in Web3”话题的讨论和分享；

这次的头脑风暴碰撞出了很多火花，我们将此次的分享整理成视频，供大家学习与交流；

https://www.youtube.com/watch?v=psdZ7XRfeS0&t=2381s

**Reply by Emilyxiannv** [reply]
我们内部做了内容创作的一个hub DAO叫dao2, 每周会翻译一些Web3，NFT，ETH生态相关的内容，以及一些原创输出的内容，具体内容参见：https://dao2.muse.lol/，有兴趣欢迎查看

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 日常志愿者翻译，L2, PoS, etc
- 优秀志愿者夏季礼物分发中

**Comment by stephanieeee-ho** [comment]
### ECN
- 日常翻译和周报编辑
- dicord E群誌的策划，已上线

**Comment by yanyanho** [comment]
# Dapp-Learning

## repo update

8个PR #782 ～ #792

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## gitcoin grant

- https://gitcoin.co/grants/3414/dapp-learning-developer-group-1

## 一周年文章
- https://mp.weixin.qq.com/s/xcH7b4jVIagoVby9Ci6-kA

## sharing meeting

成功举办 《 zkproof in mina 》技术分享会，感谢各个社区的支持


近两周举办了2场分享会

- 《SBT 的来龙去脉》by tingfei 
- 《Black-Scholes模型从入门到放弃》by Kaiji
- 《DA技术简析》 by cyberorange
-  转载《零知识证明技术分享》by 张烨& 李辰星 
 https://www.youtube.com/c/DappLearning

## 翻译小组

- 正式发布18-19集
  - https://mp.weixin.qq.com/s/XZPF7sIAckcYx8-wlobegw
